bootloader/grub.go

```diff
@@ -604,10 +604,7 @@ func (g *grub) getGrubShimBinaryFullPath() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	if osutil.FileExists(filepath.Join(g.rootdir, assets.fallbackBinary)) {
-		return filepath.Join(g.rootdir, assets.shimBinary), nil
-	}
-	return filepath.Join(g.rootdir, assets.defaultShimBinary), nil
+	return filepath.Join(g.rootdir, assets.shimBinary.path), nil
```

Comment:
What if the `shimBinary` doesn't exist and only the `defaultShimBinary` exists?

Reply:
That was intermediate. This is removed in 7be3e78303fa4273cba1ee3037db16ea01f45a60.